data/json/martialarts.json

```diff
@@ -1389,6 +1389,60 @@
     ],
     "techniques": [ "tec_tiger_palm", "tec_tiger_takedown" ]
   },
+  {
+    "type": "martial_art",
+    "id": "style_wingchun",
+    "name": "Wing Chun",
+    "description": "Wing Chun is a Chinese martial art that began by selecting the easiest-to-learn effective techniques from the various Shaolin animal forms.  It has a high stance, with weight entirely on the rear leg.  Wing Chun focuses on sensitivity to the opponent's direction of force, and flowing around it to get back to hitting.",
+    "initiate": [
+      "You take your stance and prepare to receive the gift of violence.",
+      "%s patiently assumes a curiously pigeon-toed stance."
+    ],
+    "learn_difficulty": 8,
+    "arm_block": 2,
+    "leg_block": 3,
+    "static_buffs": [
+      {
+        "id": "buff_wingchun_static",
+        "name": "Chi-Sao Sensitivity",
+        "description": "You have a greater understanding of balance and technique. This gives you a better chance to avoid your opponent's attacks.\n\n Dodging Skill increased by 15%% of Perception.  Blocked damage reduced by 50%% of Perception.",
```

Comment:
```suggestion
        "description": "You have a greater understanding of balance and technique.  This gives you a better chance to avoid your opponent's attacks.\n\n Dodging Skill increased by 15%% of Perception.  Blocked damage reduced by 50%% of Perception.",
```